CONTRIBUTING.md

```diff
@@ -55,4 +58,130 @@ We prefer all communications to be in English.
+
+- Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files. When running the check section, results will not fail a build. Check the logs for the results of this section. 
+
+- All package sources are available. The sources are either in the source server or local folders `SPECS/<package>/SOURCES` or `SPECS/<package>`. While it is possible to build packages with all sources inside the repo, our policy is generally to have the source compressed and placed on the source server. Uploading to the source server can only be accomplished by a CBL-Mariner developer. Please request help in your PR for uploading your sources to the source server. To check the source server see https://cblmarinerstorage.blob.core.windows.net/sources/core/< source tar >.
+
+- cgmanifest files are up-to-date and alphabetically sorted. The cgmanifest files are used to record all package sources. They include the following files:
+    - ./cgmanifest.json
+    - ./toolkit/tools/cgmanifest.json
+    - ./toolkit/scripts/toolchain/cgmanifest.json,
+    - .github/workflows/cgmanifest.json
+
+  To validate, run the following in a CBL-Mariner container or Ubuntu >= 22.04    
+  ```bash
+  .github/workflows/validate-cg-manifest.sh SPECS/<package name>/<package-name>.spec
+   ```
+- LICENSE-MAP files are up-to-date. These files indicate which licenses are being used by CBL-Mariner's packages and where the package may be derived from. The license files include the following files:
+  - ./SPECS/LICENSES-AND-NOTICES/data/licenses.json
+  - ./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md
+
+  This can be checked by running
+  ```bash
+  python3 ./toolkit/scripts/license_map.py  \
+  ./SPECS/LICENSES-AND-NOTICES/data/licenses.json \
+  ./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md \
+  ./SPECS \
+  ./SPECS-EXTENDED \
+  ./SPECS-SIGNED
+  ```
+- All source files have up-to-date hashes in the *.signatures.json files. This hash is generated by `sha256sum`. You can also use the make tools to automatically update these hashes. 
+   ```bash
+   cd toolkit
+   sudo make input-srpms REBUILD_TOOLS=y SRPM_FILE_SIGNATURE_HANDLING=update
+   ```
+- If you modified the go tools, ensure that the formatting meets golang's standards and that the tests still pass. 
+   ```bash
+   cd toolkit
+   # fix formatting
+   sudo make go-tidy-all
+   # check tests
+   sudo make go-test-coverage
+   ```
+- Documentation has been updated to match any changes to the build system. See [toolkit/docs](toolkit/docs).
+
+# Bugs
+If the bug is security related, please use the [security guidelines](#security-vulnerabilities) above. Please use the [issues page](https://github.com/microsoft/CBL-Mariner/issues) on CBL-Mariner to file any other bugs.
```

Comment:
```suggestion
If the bug is security related, please use the [security guidelines](#security-vulnerabilities) above. Otherwise, please use the [issues page](https://github.com/microsoft/CBL-Mariner/issues) on CBL-Mariner to file any other bugs.
```